### PR TITLE
Adding new icons for Custom Report 

### PR DESCRIPTION
**Pull request by jaiswalanjali**

### CODE REVIEW

File: packages/pebble-shared/icons.json [hunk 1]
```diff
@@ -1,4 +1,12 @@
 [
+  "Edit note",
+  "Hidden",
+  "Lock",
+  "Report",
+  "Schedule mail",
+  "System",
+  "Unlock",
+  "actions",
```
Comment (zhirzh) [inline packages/pebble-shared/icons.json:9]: icon names should be all lowercase and hyphen separated
example: "Edit note" should be "edit-note"

**Review by zhirzh — CHANGES_REQUESTED** [review]

File: packages/pebble-shared/icons.json (hunk 1 above)
Comment (jaiswalanjali) [reply packages/pebble-shared/icons.json]: done

**Review by zhirzh — APPROVED** [review]

**Review by l3satwik — CHANGES_REQUESTED** [review]
there should be no padding around the icons.

File: packages/pebble-shared/svgs/Hidden.svg [hunk 2]
```diff
@@ -1 +1 @@
...
...4 64.148 0 125.843-11.236 183.044-31.665l183.657 186.109 56.384-55.567L72.319 41l-56.18 55.771zM499.898 725.78c-173.442 0-331.154-96.016-411.849-247.395 34.934-65.781 84.985-120.94 143.82-162.615l80.082 81.103c-10.827 24.924-16.752 52.503-16.752 81.512 0 113.177 91.726 204.903 204.903 204.903 27.784 0 54.546-5.72 78.448-15.526l41.675 42.288c-38.815 10.214-79.265 15.73-120.327 15.73zM375.894 461.838l138.713 141.164c-4.903.613-9.602.817-14.709.817-69.254 0-125.434-56.18-125.434-125.434 0-5.311.613-11.032 1.43-16.547zM688.253 559.284c10.418-24.72 16.343-51.89 16.343-80.695 0-113.177-91.727-204.903-204.903-204.903-27.375 0-53.933 5.312-77.631 15.322l63.739 64.964c4.494-.613 9.193-.817 13.892-.817 69.254 0 125.434 56.18 125.434 125.434 0 5.312-.613 10.419-1.022 15.526l64.148 65.169z"/><path d="M499.898 230.786c173.442 0 331.154 96.016 411.849 247.395-34.934 65.577-84.168 120.735-143.208 162.41l56.384 57.202C900.715 641.409 961.798 565.821 1000 478.385c-84.576-192.441-276.405-326.864-500.102-326.864-64.147 0-125.434 11.032-182.431 31.256l62.921 63.739c38.611-10.01 78.652-15.73 119.51-15.73z"/></g><defs><clipPath id="clip0"><path fill="#fff" d="M0 0h1000v1000H0z"/></clipPath></defs></svg>
```
Comment (zhirzh) [inline packages/pebble-shared/svgs/Hidden.svg]: group elements black color fill (`fill="#000"`)
remove this

Comment (zhirzh) [inline packages/pebble-shared/svgs/Hidden.svg]: avoid using `clip-path` in svg. it doesnt work great on android ([see here](https://youtu.be/fgbl34me3kk?t=346))

**Review by zhirzh — CHANGES_REQUESTED** [review]

**Pull request closed without merging**